packages/rum-core/src/domain/view/viewMetrics/trackFirstInput.spec.ts

```diff
@@ -12,31 +12,23 @@ describe('firstInputTimings', () => {
   const lifeCycle = new LifeCycle()
   let fitCallback: jasmine.Spy<(firstInput: FirstInput) => void>
   let configuration: RumConfiguration
-  let cleanup: () => void
 
   function startFirstInputTracking() {
+    configuration = {} as RumConfiguration
+    fitCallback = jasmine.createSpy()
+
     const firstHidden = trackFirstHidden(configuration)
     const firstInputTimings = trackFirstInput(lifeCycle, configuration, firstHidden, fitCallback)
 
-    cleanup = () => {
+    registerCleanupTask(() => {
       firstHidden.stop()
       firstInputTimings.stop()
-    }
+      resetExperimentalFeatures()
```

Comment:
**🥜 nitpick:** ‏you could remove `resetExperimentalFeatures()` as we don't use experimental features in that test suite anymore.

Reply:
Done ✔️